modules/nw-dpu-installing-operator-cli.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/installing-dpu-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="install-operator-cli_{context}"]
+= Installing the DPU Operator by using the CLI
+
+As a cluster administrator, you can install the DPU Operator by using the CLI.
+
+[NOTE]
+====
+The CLI must be used to install the DPU Operator on the DPU cluster.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
```

Comment:
An attribute exists for OpenShift CLI: :oc-first: pass:quotes[OpenShift CLI (`oc`)]